### ISSUE TITLE
Changes not included in the tagged version

### ISSUE THREAD

**Issue by mirceaulinic** [saltstack/salt#43242]
### Description of Issue/Question

I was looking at https://github.com/saltstack/salt/blob/v2017.7.1/salt/proxy/napalm.py but I don't see the changes from https://github.com/saltstack/salt/pull/42236 although I recall the PR has been merged before the tagging started for `2017.7.1`.
Can anyone clarify please?

**Comment by rallytime** [comment]
Hi @mirceaulinic - The 2017.7.1 release was a more targeted release that contained fewer fixes. We're moving towards making release branches (2017.7.1) created from the parent branch (2017.7) to help with release stability.

I just checked and the fix you referenced is not in the 2017.7.1 release. It will be in the 2017.7.2 release.

The docs have been updated [here](https://docs.saltstack.com/en/latest/topics/development/contributing.html#release-branches) to help explain the change.
